frontend_lib/src/component/PopinFixed/PopinFixedContent.jsx

```diff
@@ -17,41 +27,169 @@ class PopinFixedContent extends React.Component {
   }
 
   render () {
-    return this.props.children.length === 2
+    const { props } = this
+    return props.children.length === 2
```

Comment:
Instead of using a ternary operator, would using an early return be easier to read?
```javascript
if (props.children.length !== 2) {
  return (
     <div className={classnames('wsContentGeneric__content', `${props.customClass}__content`)}>
          {props.children}
        </div>
      )
}
```

Reply:
Also as the behavior with 2 children is totally different I would have expected two components if the number of children is static (maybe a technical debt issue to write?).

Reply:
This code was not done in this PR (just added the `const { props } = this`), I will write an issue about it.

Reply:
OK.

Reply:
https://github.com/tracim/tracim/issues/4750